app/server/api/credit_transfer_api.py

```diff
@@ -342,15 +351,13 @@ def post(self, credit_transfer_id):
 
             else:
                 message = 'Transfer Successful' if auto_resolve else 'Transfer Pending. Must be approved.'
-
+                db.session.commit()
```

Comment:
Have to commit here since on big disbursements, these rows have to be in the database by the time the callback comes